src/fitnesse/slim/converters/PropertyEditorManager.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Why this copyright message?

Reply:
The PropertyEditorManager was copied over from original source in ASF. I'm open for suggestion what to put in here.

Reply:
If there are no modifications, just leave it as-is. Otherwise add a line to indicate copyright of FitNesse contributors.